src/components/ListItemAccount.vue

```diff
@@ -39,7 +44,7 @@ export default {
     subtitleContent() {
       switch (this.subtitle) {
         case 'balance':
-          return this.balance ? `${prefixedAmount(this.balance)} AE` : '';
+          return '';
```

Comment:
Let's apply the conversion to currency here instead of passing Balance to subtitle